tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_new_window.py

```diff
@@ -0,0 +1,56 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.search.app import Search
+from gaiatest.apps.system.regions.cards_view import CardsView
+
+
+class TestBrowserNewWindow(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+        self.test_url = self.marionette.absolute_url('mozilla.html')
+        self.test_url2 = self.marionette.absolute_url('mozilla_community.html')
+
+    def test_browser_new_window(self):
+        search = Search(self.marionette)
+        search.launch()
+        browser = search.go_to_url(self.test_url)
+        browser.wait_for_page_to_load()
+
+        browser.tap_menu_button()
+        browser.open_new_window()
+        search.set_root_element()
+        search.wait_to_be_displayed()
+        browser2 = search.go_to_url(self.test_url2)
+        browser2.wait_for_page_to_load()
+        browser2.switch_to_content()
+        self.assertEqual('Mozilla Community', self.marionette.title)
+        browser2.switch_to_chrome()
+
+        cards_view = CardsView(self.marionette)
+        self.device.hold_home_button()
+        cards_view.wait_for_cards_view()
+        cards_view.cards[1].wait_for_centered()
```

Comment:
When I take a look at the animation, the card view is centered from the beginning to the end of the animation. We can wait for the unzoom effect to be over, though.

To do so, I'd suggest to look in the style and make sure that there is no `transition` in the style anymore. Each time a card moves, you can see a CSS `transition` defined